cypress/e2e/hermioneAccount.spec.cy.js

```
@@ -0,0 +1,87 @@
+/// <reference types='cypress' />
+
+import { faker } from '@faker-js/faker';
+
+describe('Bank app', () => {
+  const user = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  let initialBalance;
+
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.login(user);
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '5096')
```

Using a hardcoded value like '5096' in the selector makes the test brittle. If the initial balance changes, this selector will fail. It's better to locate the element without relying on its content. For example, you can find the element containing the text 'Balance' and then get the `<strong>` element inside it to read the value.